src/gateway/get-started/services-and-routes.md

```diff
@@ -0,0 +1,455 @@
+complete that before proceeding.
+
+### Managing services
+
+1. **Creating services**
+
+   To add a new service, send a `POST` request to {{site.base_gateway}}'s 
+   Admin API `/services` route:
+
+   ```sh
+   curl -i -s -X POST http://localhost:8001/services \
+     --data name=example_service \
+     --data url='http://mockbin.org'
+   ```
+
+   This request instructs {{site.base_gateway}} to create a new 
+   service mapped to the upstream URL `http://mockbin.org`.
+   
+   In our example, the request body contained two strings: 
+   
+   * `name`: The name of the service
+   * `url` : An argument that populates the `host`, `port`, and `path` attributes of the service
+   
+   If your request was successful, you will see a `201` response header from {{site.base_gateway}} 
+   confirming that your service was created and the response body will be similar to:
+
+   ```text
+   {
+     "host": "mockbin.org",
+     "name": "example_service",
+     "enabled": true,
+     "connect_timeout": 60000,
+     "read_timeout": 60000,
+     "retries": 5,
+     "protocol": "http",
+     "path": null,
+     "port": 80,
+     "tags": null,
+     "client_certificate": null,
+     "tls_verify": null,
+     "created_at": 1661346938,
+     "updated_at": 1661346938,
+     "tls_verify_depth": null,
+     "id": "3b2be74e-335b-4f25-9f08-6c41b4720315",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
+   ```
+   
+   Fields that are not explicitly provided in the create request are automatically given 
+   a default value based on the current {{site.base_gateway}} configuration.  
+   The [Admin API documentation](/gateway/latest/admin-api/#service-object) has the 
+   full specification for managing service objects.
+
+1. **Viewing service configuration**
+
+   When you create a service, {{site.base_gateway}} assigns it a unique `id` as shown in the response above. 
+   The `id` field, or the name provided when creating the service, can be used to identify the service 
+   in subsequent requests. This is the service URL and takes the form of `/services/{service name or id}`.
+
+   To view the current state of a service, make a `GET` request on the service
+   URL.
+
+   ```sh
+   curl -X GET http://localhost:8001/services/example_service
+   ```
+   
+   The response body contains the current configuration of your service: 
+   
+   ```text
+   {
+     "host": "mockbin.org",
+     "name": "example_service",
+     "enabled": true,
+     "connect_timeout": 60000,
+     "read_timeout": 60000,
+     "retries": 5,
+     "protocol": "http",
+     "path": null,
+     "port": 80,
+     "tags": null,
+     "client_certificate": null,
+     "tls_verify": null,
+     "created_at": 1661290122,
+     "updated_at": 1661290122,
+     "tls_verify_depth": null,
+     "id": "c1e98b2b-6e77-476c-82ca-a5f1fb877e07",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
+   ```
```

Comment:
I think this one can probably be replaced with a variation of "If the request was successful you will receive a `200` response and a response body containing the current configuration."

Reply:
I do think the point of this step is to show how to view the configuration, so I would prefer to keep this here, but I can shorten what's shown and use a snippet of the important parts instead to reduce the length

Reply:
Please see latest